main.cpp

```diff
@@ -1 +1,238 @@
+#include <iostream>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+#include <vector>
+#include <charconv>
+#include <algorithm>
+#include <iterator>
+#include <stdio.h>
+
+using namespace std;
+
+void print(int guesses = 0) { // print the hangman
```

Comment:
At Immersive (and in general C++ devs) put the bracket at new line. But you do what you prefer.
Just remember that Immersive standard is different